src/ResourceManager/Storage/Commands.Management.Storage/help/Get-AzureRmStorageAccount.md

```diff
@@ -91,7 +91,7 @@ Type: System.String
 Parameter Sets: ResourceGroupParameterSet
 Aliases:
 
-Required: False
```

Comment:
The place you changed is in "ResourceGroupParameterSet ", which means don't give storage account name. In this case, when give Resouce Group name, will list account in this resource group ; when don't give resource group name, will list all account in the subscription. So the Resouce Group name is not required, and the change is not right.